src/partials/external_authentication_failed.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="card">
+  <div class="org-logo">
+    <img class="org-logo__image" src="{{logo}}" alt="Company Logo" />
+  </div>
+  <h1 class="heading">Authentication Failed!</h1>
+  {{#if userMessage}}
+    <div class="text-block">
```

Comment:
Do we want to use the {{> errors}} handlebar for displaying the errors?

Reply:
ah, yes...didn't see that one there. Thanks